internal/server/errors.go

```diff
@@ -39,6 +39,8 @@ func sendAPIError(c *gin.Context, err error) {
 		resp.Code = http.StatusUnauthorized
 		// hide the error text, it may contain sensitive information
 		resp.Message = "unauthorized"
+		// log the error at warn, since it is not in the response
+		log = logging.L.WithOptions(zap.AddCallerSkip(1)).Warn
```

Comment:
Seems more like an info log than a warn log?

Reply:
Ya, info would probably be fine too. My thinking with warning is that someone who wants to hide the request logs that are at INFO level might set their logging to WARN, and then lose the ability to debug any authn errors.  But maybe that's an unlikely scenario. Let's see if anyone else has thoughts on warn vs info.

Reply:
I would personally prefer info for the reason that any outside party can trigger this log by attempting to login with invalid an username/password, which could flood to logs.

Reply:
I changed it to INFO. It might be interesting to see if we can combine this logging of the error with the request log. I think they are effectively logging the same event, so it would be cool if they were fields on the same log line. I'll probably look into that as a follow up.